livekit-android-sdk/src/main/java/io/livekit/android/webrtc/PeerConnectionExt.kt

```diff
@@ -6,10 +6,13 @@ import org.webrtc.PeerConnection
  * Completed state is a valid state for a connected connection, so this should be used
  * when checking for a connected state
  */
-internal fun PeerConnection.isConnected(): Boolean {
-    return when (iceConnectionState()) {
+internal fun PeerConnection.isConnected(): Boolean = iceConnectionState().isConnected()
```

Comment:
I think @cnderrauber moved to using PeerConnection state is JS SDK as one true indicator of connection state. He also moved to it on server side.

Can we do that in the mobile SDKs also?

Reply:
Switch from `iceConnectionState` to `peerConnectionState` throughout the SDK ?

Reply:
yeah, IceConnected not means PeerConnection is connected, because there is a case that ICE connected but DTLS failed, then PeerConnection can't transit to connect